content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,282 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
 needed. Scheduling repairs and managing services for an entire fleet is necessary to ensure all
 vehicles are in good working order when they are needed.
 
-Create a service record
-=======================
+.. _service/service-form:
```

Comment:
Please include the app/module name in the custom anchor. Please make sure to update all custom anchors and :ref: tags :) 
https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#prefix-hyperlink-targets-with-application-names
```suggestion
.. _fleet/service-form:
```